tests/data/melodymodel/5_2/Melody Model Test.capella

```diff
@@ -1379,6 +1379,20 @@ The predator is far away</bodies>
                   sourceElement="#0fef2887-04ce-4406-b1a1-a1b35e1ce0f3"/>
             </ownedClasses>
           </ownedDataPkgs>
+          <ownedDataPkgs xsi:type="org.polarsys.capella.core.data.information:DataPkg"
+              id="80b36e27-cf57-44e6-862b-37f52b77bcc8" name="TestTrace">
+            <ownedClasses xsi:type="org.polarsys.capella.core.data.information:Class"
+                id="ed272baf-43f2-4fa1-ad50-49c00563258b" name="Class Trace"/>
```

Comment:
This name is a bit confusing, this isn't a trace by itself but rather a class that acts as target for a trace.